### ISSUE TITLE
How to disable tabs switching?

### ISSUE THREAD

**Issue by aglaganov** [uikit/uikit#927]
How can I make tabs with responsiveness but without switching?

**Comment by jerry1970** [comment]
Can you be more clear on what you want to achieve?

**Comment by aheinze** [comment]
could you please be more detailed?

**Comment by aglaganov** [comment]
http://getuikit.com/docs/tab.html
If I will use data-uk-tab attribute, there will be responsiveness with switching.
If I will use data-uk-switcher attribute, there will be switching but without responsiveness.
I need responsiveness but without switching, is it possible?

**Comment by jerry1970** [comment]
Still not sure what the responsiveness means, but I assume you want to have tabs that work like tabs (so click on a tab makes it active) but do nothing else. When I use the first example from the link you provide, I get exactly that:

``` html
<ul class="uk-tab my-tab" data-uk-tab>
    <li class="uk-active"><a href="">Tab 1</a></li>
    <li><a href="">Another tab</a></li>
    <li><a href="">Last tab</a></li>
</ul>
```

Don't add anything else, just that. Well, I assume you want to do something when the user clicks on one of the tabs, so you would probably add your own listeners:
``` javascript
$(function(){
    $('.my-tab li').on('click', function(event) {
        console.debug('This is the LI that was clicked:', $(event.target));
    });
});
```

If this is not what you want, please explain your problem without using the word 'responsive' because that is causing some confusion.

**Comment by aglaganov** [comment]
I need responsiveness, when all tabs collapse to 1 menu on small screens.
But on click I want to go to another page by href

**Comment by jerry1970** [comment]
I suggested you would rephrase your request without using the word 'responsiveness' because it means too much.

The second sentence is much clearer: click on a tab should go to another URL. Do you want that all the time or only when the tabs are collapsed into a menu? If you want it only when the tabs are collapsed into a menu, I'd say you are wanting something different than tabs. Why would you want that?

You could add listeners to tabs based on ```.uk-tabs > li > a``` or ```uk-tabs-responsive > li > a```, but you'd have to listen to a page resize as well. Way too difficult. Stick to the regular tab control or do something completely different like show tabs on medium or larger screens and a dropdown button with menu items on a small screen. Then you have the default tab function on medium and larger screens and a default dropdown button on a small screen. Yes, that's code duplication but a lot less code than adding listeners.

**Comment by jlove73071** [comment]
I've got the same issue:
What I want is the menu to collapse and DISPLAY just like it does with tabs in that the menu collapses on smaller screens.. I, however, want the clicking to navigate to another URL.. so I'm using it as a sort of "tabbed menu" in my example, not a single item will use the default tab switching, they will all just navigate to the URL in the item.
